molecularnodes/nodes/styles.py

```diff
@@ -0,0 +1,281 @@
+        Socket(name="backbone_smoothing", blendername="Backbone Smoothing"),
+        Socket(name="backbone_threshold", blendername="Backbone Threshold"),
+        Socket(name="backbone_radius", blendername="Backbone Radius"),
+        Socket(name="backbone_shape", blendername="Backbone Shape"),
+        Socket(name="base_scale", blendername="Base Scale"),
+        Socket(name="base_resolution", blendername="Base Resolution"),
+        Socket(name="base_realize", blendername="Base Realize"),
+        Socket(name="uv_map", blendername="UV Map"),
+        Socket(name="u_component", blendername="U Component"),
+    ]
+
+    def __init__(
+        self,
+        quality: int = 3,
+        color_blur: bool = False,
+        shade_smooth: bool = False,
+        backbone_smoothing: float = 0.5,
+        backbone_threshold: float = 4.5,
+        base_scale: Tuple[float, float, float] = (2.5, 0.5, 7.0),
+        backbone_radius: float = 2.0,
+        backbone_shape: str = "Cylinder",
+        base_resolution: int = 4,
+        base_realize: bool = False,
+        uv_map: bool = False,
+        u_component=None,
+        # u_component_factor=None,
+    ):
+        self.quality = quality
+        self.color_blur = color_blur
+        self.shade_smooth = shade_smooth
+        self.backbone_smoothing = backbone_smoothing
+        self.backbone_threshold = backbone_threshold
+        self.backbone_radius = backbone_radius
+        self.backbone_shape = backbone_shape  # enum?
+        self.base_scale = base_scale
+        self.base_resolution = base_resolution
+        self.base_realize = base_realize
+        self.uv_map = uv_map
+        self.u_component = u_component
+
+
+class StyleSpheres(StyleBase):
+    style = "spheres"
+    portdata: SocketInfo = [
+        Socket(name="geometry", blendername="Sphere Geometry"),
+        Socket(name="radii", blendername="Sphere Radii"),
+        Socket(name="sphere_subdivisions", blendername="Sphere Subdivisions"),
+        Socket(name="shade_smooth", blendername="Shade Smooth"),
+    ]
+
+    def __init__(
+        self,
+        geometry: str = "Point",  # enum: "Point" (Point Cloud), "Instances" (Instances of a mesh Icosphere), or "Mesh" (realised Mesh)
+        radii: float = 0.8,
+        sphere_subdivisions: int = 2,
+        shade_smooth: bool = True,
+    ):
+        self.geometry = geometry
+        self.radii = radii
+        self.sphere_subdivisions = sphere_subdivisions
+        self.shade_smooth = shade_smooth
+
+
+class StyleSticks(StyleBase):
+    style = "sticks"
+    portdata: SocketInfo = [
+        Socket(name="quality", blendername="Quality"),
+        Socket(name="radius", blendername="Radius"),
+        Socket(name="color_blur", blendername="Color Blur"),
+        Socket(name="shade_smooth", blendername="Shade Smooth"),
+    ]
+
+    def __init__(
+        self,
+        quality: int = 3,
+        radius: float = 0.20000000298023224,
```

Comment:
I know inside of blender it's technically the floating point version but in code we can just have it be 0.2

Reply:
good catch